stable/openvpn/Chart.yaml

```
@@ -3,7 +3,7 @@ description: A Helm chart to install an openvpn server inside a kubernetes clust
   generation is also part of the deployment, and this chart will generate client keys
   as needed.
 name: openvpn
-version: 3.11.0
+version: 3.11.1
```

adding new functionality 
```suggestion
version: 3.12.0
```